public/plugins/ckeditor/skins/bootstrapck/scss/browser-specific/ie8/dialog_ie8.scss

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2003-2014, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+/*
+dialog_ie8.css
+===============
+
+This file contains styles to used by Internet Explorer 8 only.
+*/
+
+/* Base it on dialog_ie.css, overriding it with styles defined in this file. */
+@import "../../dialog/dialog";
+
+/*  Without the following, IE8 cannot compensate footer button thick borders
+    on :focus/:active. */
+a.cke_dialog_ui_button_ok:focus span,
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `cke_dialog_ui_button_ok` should be written in hyphenated BEM (Block Element Modifier) format<br>Selector `cke_dialog_ui_button_cancel` should be written in hyphenated BEM (Block Element Modifier) format